packages/valory/services/trader/service.yaml

```diff
@@ -154,8 +154,9 @@ type: skill
         collateral_balance: ${BENCHMARKING_MODE_COLLATERAL_BALANCE:int:10000000000000000000}
         mech_cost: ${BENCHMARKING_MODE_MECH_COST:int:10000000000000000}
         pool_fee: ${BENCHMARKING_MODE_POOL_FEE:int:20000000000000000}
-        outcome_token_amounts: ${OUTCOME_TOKEN_AMOUNTS:list:[11000000000000000000,9000000000000000000]}
-        outcome_token_marginal_prices: ${OUTCOME_TOKEN_MARGINAL_PRICES:list:[0.55,0.45]}
+        outcome_token_amounts: ${OUTCOME_TOKEN_AMOUNTS:dict:{"1":[11000000000000000000,9000000000000000000],"2":[12051566363198041123,4065861525654596129],"3":[7873230340650588731,6223620785868051083]}}
+        outcome_token_marginal_prices: ${OUTCOME_TOKEN_MARGINAL_PRICES:dict:{"1":[0.39537,0.60462],"2":[0.252264,0.747735],"3":[0.44149,0.5585]}}
+        scaled_liquidity_measure: ${SCALED_LIQUIDITY_MEASURE:dict:{"1":9.79065,"2":6.08036330493113,"3":6.951844672001134}}
```

Comment:
Forgot to remove these initial default values. They do not make anymore sense in the new logic since the number of entries depends on the number of markets to simulate and that can be different in each experiment.